### PR TITLE
Adding second data series to big graph

### PR DESCRIPTION
**Pull request by WojcikMike**

### CODE REVIEW

File: src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js [hunk 1]
```diff
@@ -1,110 +1,151 @@
...
+                                height,
+                                graphWidth,
+                                margin,
+                                dates,
+                                'queue-length');
+
+                            chart.append("text")
+                                .attr("text-anchor", "middle")
+                                .attr("transform", "translate(" + (margin / 2) + "," + (height / 2) + ")rotate(-90)")
+                                .text("Throughput [msgs/s]");
```
Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js]: @WilliamBZA do we want to parametrize this value? At the moment label is hard coded, as well as a class name. I could add it to the data set passed to the graph, and populate the values in a controller. What do you think?

Comment (WojcikMike) [reply src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js]: I decided to do that

**Comment by WojcikMike** [comment]
@WilliamBZA I think this is ready to be merged

File: src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js [hunk 2]
```diff
@@ -1,110 +1,148 @@
...
+
+        chart.append('text')
+            .attr('x', 0)
+            .attr('y', scaleYValue)
+            .attr('text-anchor', 'start')
+            .attr('font-size', 12)
+            .attr('font-family', 'sans-serif')
+            .text('ø');
+
+        chart.append("text")
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js]: Single quotes

File: src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js [hunk 3]
```diff
@@ -1,110 +1,148 @@
...
         .directive('largeGraph',
             function() {
                 return {
                     restrict: 'E',
-                    scope: {
-                        data: '=plotPoints',
-                        avg: '=plotAverage',
+                    scope: {                        
                         dates: '=xaxisPoints',
+                        firstDataSeries: '=firstDataSeries',
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js]: Is it not better to accept an array of data series?

Comment (WojcikMike) [reply src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js]: for now the graph is designed to display 2 series one of the reasons is that we display up to 2 Y axis (one on the left and one of the right). I don't know how to handle this in a generic way

**Pull request merged**